physics/GFS_MP_generic_post.meta

```diff
@@ -278,6 +278,76 @@
   type = real
   kind = kind_phys
   intent = inout
+[frzr]
+  standard_name = lwe_thickness_of_sfc_freezing_rain_amount
+  long_name = accumulated surface freezing rain
+  units = m
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = inout
+[frzrb]
+  standard_name = lwe_thickness_of_sfc_freezing_rain_amount_in_bucket
+  long_name = accumulated surface freezing rain in bucket
+  units = m
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = inout
+[frozr]
+  standard_name = lwe_thickness_of_sfc_graupel_amount
+  long_name = accumulated surface graupel
+  units = m
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = inout
+[frozrb]
+  standard_name = lwe_thickness_of_sfc_graupel_amount_in_bucket
+  long_name = accumulated surface graupel in bucket
+  units = m
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = inout
+[tsnowp]
+  standard_name = lwe_thickness_of_sfc_snow_amount
+  long_name = accumulated surface snow
+  units = m
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = inout
+[tsnowpb]
+  standard_name = lwe_thickness_of_sfc_snow_amount_in_bucket
+  long_name = accumulated surface snow in bucket
+  units = m
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = inout
+[rhonewsn1]
+  standard_name = lwe_density_of_precip_ice
```

Comment:
we've been spelling out `precipitation` to be consistent rather than `precip` or `prcp` below.